v2.1/reference/tidb-binlog/faq.md

```diff
@@ -1,13 +1,143 @@
 ---
 title: TiDB Binlog FAQ
 summary: Learn about the frequently asked questions (FAQs) and answers about TiDB Binlog.
-reference: faq
+category: faq
 ---
 
 # TiDB Binlog FAQ
 
 This document collects the frequently asked questions (FAQs) about TiDB Binlog.
 
+## What is the impact of enabling TiDB Binlog on the performance of TiDB?
+
+- There is no impact on the query.
+
+- There is a slight performance impact on `INSERT`, `DELETE` and `UPDATE` transactions. In latency, a p-binlog is written concurrently in the TiKV prewrite stage before the transactions are committed. Generally, writing binlog is faster than TiKV prewrite, so it does not increase latency. You can check the response time of writing binlog in Pump's monitoring panel.
+
+## What privileges does Drainer need to replicate data to the downstream MySQL or TiDB cluster?
+
+To replicate data to the downstream MySQL or TiDB cluster, Drainer must have the following privileges:
+
+* Insert
+* Update
+* Delete
+* Create
+* Drop
+* Alter
+* Execute
+* Index
+* Select
+
+## What can I do if the Pump disk is almost full?
+
+1. Check whether Pump's GC works well:
+
+    - Check whether the **gc_tso** time in Pump’s monitoring panel is identical with that of the configuration file. For Pump of version <= v2.1.14, GC works after the non-offline drainer consumes data. If there are drainer instances that are no longer in use, use binlogctl to make them offline.
```

Comment:
This line is different for different versions. PTAL @july2993 @lilin90 Thanks~

Reply:
LGTM